src/client/mod.rs

```diff
@@ -122,12 +126,25 @@ impl Client {
         let inner = Arc::new(ClientInner {
             topology: Topology::new(options.clone())?,
             session_pool: ServerSessionPool::new(),
+            #[cfg(feature = "fle")]
+            fle: Default::default(),
             options,
         });
-
         Ok(Self { inner })
     }
 
+    /// Creates a new `Client` connected to the cluster specified by `options` with auto-encryption
+    /// enabled.
+    #[cfg(feature = "fle")]
+    pub async fn with_fle_options(
```

Comment:
related to my question above, maybe this would be better named `with_encryption_opts`?

Reply:
Done.